### DISCUSSION TITLE
Integration of SASS

### DISCUSSION THREAD

**Discussion by sonnenblume007** [varabyte/kobweb#628] [Q&A] [answered]
Hi
Is it possible to work with scss? Can somebody explain me how I can integrate it in a kobweb project?
Thanks for your help!

**Comment by bitspittle** [comment, marked as answer]
I don't think anyone has tried it yet. I imagine you'd install it as a dev npm dependency in your build script and then add a gradle task to run it, processing any scss or sass files under your resource directory?

However, I highly recommend you just work with Style blocks for now. You don't need the programmatic support of sass when you can already lean on the richer programmatic support from Kotlin!

If you give me a specific example of something you want to do in an scss file, I can try to show you the equivalent Kobweb code.

**Reply by sonnenblume007** [reply]
Hey thanks for your answer. I will try it with style blocks.
